test/functional/test_framework/bip324_cipher.py

```diff
@@ -112,3 +161,24 @@ def test_aead(self):
             self.assertEqual(hex_cipher, ciphertext.hex())
             plaintext = aead_chacha20_poly1305_decrypt(key, nonce, aad, ciphertext)
             self.assertEqual(plain, plaintext)
+
+    def test_fschacha20poly1305aead(self):
+        "FSChaCha20Poly1305 AEAD test vectors."
+        for test_vector in FSAEAD_TESTS:
+            hex_plain, hex_aad, hex_key, msg_idx, hex_cipher = test_vector
+            plain = bytes.fromhex(hex_plain)
+            aad = bytes.fromhex(hex_aad)
+            key = bytes.fromhex(hex_key)
+
+            enc_aead = FSChaCha20Poly1305(key)
+            dec_aead = FSChaCha20Poly1305(key)
+
+            for _ in range(msg_idx):
+                enc_aead.crypt(b"", b"", False)
```

Comment:
nit: named arguments for the boolean here and below would be nice
```suggestion
                enc_aead.crypt(b"", b"", is_decrypt=False)
```